rules/windows/builtin/security/win_security_successful_external_remote_smb_login.yml

```diff
@@ -25,23 +25,18 @@ detection:
         EventID: 4624
         LogonType: 3
     filter_ipv4:
-        SubjectUserName: '-'
         IpAddress|cidr:
-            - 10.0.0.0/8
-            - 172.16.0.0/12
-            - 192.168.0.0/16
-            - 224.0.0.0/4
-            - 127.0.0.0/8
+            - '127.0.0.0/8'
+            - '10.0.0.0/8'
+            - '172.16.0.0/12'
+            - '192.168.0.0/16'
     filter_ipv6:
-        SubjectUserName: '-'
-        IpAddress|startswith:
-            - '::1'  # IPv6 loopback variant
-            - '0:0:0:0:0:0:0:1'  # IPv6 loopback variant
```

Comment:
I understand that you don't see this variant, but it is a valid representation of the loopback address. So why remove it? It does not hurt?